scripts/forking/template/whitelabel/transactions.py

```diff
@@ -0,0 +1,42 @@
+from eth_keys.datatypes import PrivateKey
+from eth_typing import Address
+
+from eth.vm.forks.petersburg.transactions import (
+    PetersburgTransaction,
+    PetersburgUnsignedTransaction,
+)
+
+from eth._utils.transactions import (
+    create_transaction_signature,
+)
+
+
+class IstanbulTransaction(PetersburgTransaction):
+    @classmethod
+    def create_unsigned_transaction(cls,
+                                    *,
+                                    nonce: int,
+                                    gas_price: int,
+                                    gas: int,
+                                    to: Address,
+                                    value: int,
+                                    data: bytes) -> 'IstanbulUnsignedTransaction':
+        return IstanbulUnsignedTransaction(nonce, gas_price, gas, to, value, data)
+
+
+class IstanbulUnsignedTransaction(PetersburgUnsignedTransaction):
+    def as_signed_transaction(self,
+                              private_key: PrivateKey,
+                              chain_id: int=None) -> IstanbulTransaction:
```

Comment:
Flake8 wants this to be:
```suggestion
                              chain_id: int = None) -> IstanbulTransaction:
```